index.js

```diff
@@ -63,6 +63,14 @@ class ServerlessCustomDomain {
     });
   }
 
+  setGivenDomainName() {
+    /**
+     * Since serverless version 1.13.0, cli variables in the serverless file
+     * are not converted until before deploy initialize.
+      */
+    this.givenDomainName = this.serverless.service.custom.customDomain.domainName;
```

Comment:
Why not just use a local variable in each of the functions instead of making this function and field?